backend/src/apiserver/storage/experiment_store_test.go

```diff
@@ -50,14 +50,14 @@ func TestListExperiments_Pagination(t *testing.T) {
 		CreatedAtInSec: 1,
 		Name:           "experiment1",
 		Description:    "My name is experiment1",
-		StorageState:   "STORAGESTATE_AVAILABLE",
+		StorageState:   "AVAILABLE",
```

Comment:
Note that this is a behavior change at storage layer for v1 experiments.
If user downgrade to an older version of api-server, it will not be able to parse this storage state string. 
That being said, downgrade might not be a support path for us--there could be other incompatibility issues. So it's probably fine to make this change to be aligned with AIP convention for enum proto.